common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp

```diff
@@ -731,6 +735,63 @@ geometry_msgs::msg::Pose calcInterpolatedPose(
 
   return output_pose;
 }
+
+inline geometry_msgs::msg::Vector3 getVector3(const double x, double y, double z)
+{
+  return geometry_msgs::build<geometry_msgs::msg::Vector3>().x(x).y(y).z(z);
+}
+
+inline geometry_msgs::msg::Twist getTwist(const geometry_msgs::msg::Vector3 & velocity)
```

Comment:
Where is this function used?
It seems `angular` is missing. I think it's not good to use incomplete messages.

Reply:
Same for `getTwistWithCovariance` and `getTwistWithCovariance`.

Reply:
These function are transported from the package related to radar_msgs (https://github.com/ros-perception/radar_msgs/blob/ros2/msg/RadarTrack.msg), which doesn't have twist angular.
But your indication is correct, so I'll fixed it.

Reply:
fixed at [9da878d](https://github.com/autowarefoundation/autoware.universe/pull/2049/commits/9da878d558912a9b61c090af9f7d52e803fad5f2)